demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -861,16 +865,26 @@ def _collect_classifiers_dependencies(pack_classifiers: list, id_set: dict, verb
             classifier_data = next(iter(classifier.values()))
             classifier_dependencies = set()
 
-            related_incident_types = classifier_data.get('incident_types', [])
-            packs_found_from_incident_types = PackDependencies._search_packs_by_items_names(
-                related_incident_types, id_set['IncidentTypes'], exclude_ignored_dependencies)
+            related_types = classifier_data.get('incident_types', [])
 
-            # classifiers dependencies from incident types should be marked as optional unless CommonTypes pack,
-            # as customers do not have to use the OOTB mapping.
-            if packs_found_from_incident_types:
-                pack_dependencies_data = PackDependencies._update_optional_commontypes_pack_dependencies(
-                    packs_found_from_incident_types)
-                classifier_dependencies.update(pack_dependencies_data)
+            if classifier_data.get('definitionId') not in [None, 'incident', 'indicator']:
```

Comment:
apparently, when `definitionId` is not filled it holds an empty string. let's handle it as well by checking:
`if classifier_data.get('definitionId') and classifier_data.get('definitionId') not in ['incident', 'indicator']`

Same comment to all additional checks of the same sort.